CourseApp.Tests/TestDate.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Xunit;
+using CourseApp;
+
+namespace CourseApp.Tests
+{
+    public class TestDate
+    {
+        private readonly Date dr = new Date();
+
+        [Fact]
+        public void Test_data_20180101()
+        {
+            DateTime hbd = new DateTime(2018, 1, 1);
+            Assert.Equal(@"Year: 01  Month: 02  Day: 01 ", dr.Vozrost(hbd));
```

Comment:
DateTime to = new DateTime(2018, 1, 1);
            DateTime form = new DateTime(2008, 1, 1);
            Assert.Equal(@"Year: 01  Month: 02  Day: 01 ", dr.Vozrost(from, to);